mnd/mnd/models.py

```diff
@@ -100,3 +105,26 @@ class PreferredContact(models.Model):
     phone = models.CharField(max_length=30, null=True, blank=True)
     email = models.CharField(max_length=30, null=True, blank=True)
     contact_method = models.CharField(choices=CONTACT_METHOD_CHOICES, max_length=30)
+
+
+class CarerRegistration(models.Model):
+
+    CREATED = 'created'
+    REGISTERED = 'registered'
+
+    REGISTRATION_STATUS_CHOICES = [
+        (CREATED, CREATED),
+        (REGISTERED, REGISTERED),
+    ]
+    carer = models.ForeignKey(PrimaryCarer, on_delete=models.CASCADE, null=False, blank=False)
```

Comment:
For this line and the following lines: `null=False, blank=False` are the default, so for consistency can we just leave them out?

Reply:
Updated.